### PR TITLE
Moved all DAL fetchings to appropriate helper class.

### PR DESCRIPTION
**Pull request by JeffreyDevloo**
When a DAL change has happened, only helpers would need changing.

### CODE REVIEW

File: ovs/extensions/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
```
Comment (JeffreyDevloo) [inline ovs/extensions/__init__.py]: This init must be removed.

File: ovs/extensions/healthcheck/helpers/storagerouter.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+    @staticmethod
+    def get_by_machine_id(machine_id):
+        """
+        Fetch a dal machine by id
+        :param machine_id: id of the machine
+        :return:
+        """
+
+        return StorageRouterList.get_by_machine_id(machine_id)
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/storagerouter.py]: no new line at end of file

File: ovs/extensions/healthcheck/helpers/storagerouter.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        :type storagerouter_ip: str
+        :return: storagerouter object
+        :rtype: ovs.dal.hybrids.storagerouter.StorageRouter
+        """
+        return StorageRouterList.get_by_ip(storagerouter_ip)
+
+    @staticmethod
+    def get_disks_by_ip(storagerouter_ip):
+        """
+
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/storagerouter.py]: add info about the method

File: ovs/extensions/healthcheck/helpers/storagerouter.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    cache_timeout = 60
+    disk_map_cache = {}
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def get_storagerouter_guid_by_ip(storagerouter_ip):
+        """
+
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/storagerouter.py]: add info about the method

File: ovs/extensions/healthcheck/helpers/storagerouter.py [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+        :type storagerouter_ip: str
+        :return: storagerouter guid
+        :rtype: str
+        """
+        return StoragerouterHelper.get_storagerouter_by_ip(storagerouter_ip).guid
+
+    @staticmethod
+    def get_storagerouter_by_ip(storagerouter_ip):
+        """
+
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/storagerouter.py]: add info about the method

File: ovs/extensions/healthcheck/helpers/storagedriver.py [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+    @staticmethod
+    def get_by_storagedriver_id(storagedriver_id):
+        """
+        Returns the storage ip of the Storagedriver
+        :param storagedriver_id: id of the storagedriver
+        :type str
+        :return:
+        """
+        return StorageDriverList.get_by_storagedriver_id(storagedriver_id)
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/storagedriver.py]: no new line at end of file

File: ovs/extensions/healthcheck/helpers/service.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+
+class ServiceHelper(object):
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def get_services():
+        return ServiceList.get_services()
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/service.py]: no new line at end of file

File: ovs/extensions/healthcheck/helpers/backend.py [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+        :return: Backend object
+        :rtype: ovs.dal.hybrids.backend.Backend
+        """
+
+        return BackendList.get_by_name(backend_name)
+
+    @staticmethod
+    def get_backend_status_by_name(backend_name):
+        """
+
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/backend.py]: add info about the method

File: ovs/extensions/healthcheck/helpers/backend.py [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+                    if alba_backend.name == albabackend_name][0]
+        except IndexError:
+            error_msg = "No Alba backend found with name: {0}".format(albabackend_name)
+            BackendHelper.LOGGER.error(error_msg)
+            raise NameError(error_msg)
+
+    @staticmethod
+    def get_asd_safety(albabackend_guid, asd_id, api):
+        """
+        Request the calculation of the disk safety
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/backend.py]: add space between info and param.

File: ovs/extensions/healthcheck/helpers/backend.py [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+        :return: backend_guid
+        :rtype: str
+        """
+
+        return BackendList.get_by_name(backend_name).guid
+
+    @staticmethod
+    def get_backend_by_name(backend_name):
+        """
+
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/backend.py]: add info about the method

File: ovs/extensions/healthcheck/helpers/backend.py [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+        :return: alba_backend_guid
+        :rtype: str
+        """
+
+        return BackendList.get_by_name(backend_name).alba_backend_guid
+
+    @staticmethod
+    def get_backend_guid_by_name(backend_name):
+        """
+
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/backend.py]: add info about the method

File: ovs/extensions/healthcheck/helpers/backend.py [hunk 12]
```diff
@@ -0,0 +1,212 @@
...
+    """
+    LOGGER = LogHandler.get(source='helpers', name="ci_backend")
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def get_alba_backend_guid_by_name(backend_name):
+        """
+
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/backend.py]: add info about the method

File: ovs/extensions/healthcheck/helpers/backend.py [hunk 13]
```diff
@@ -0,0 +1,212 @@
...
+        :return: task id of celery
+        :rtype: str
+        """
+        params = {'asd_id': asd_id}
+        return api.get('alba/backends/{0}/calculate_safety'.format(albabackend_guid), params=params)
+
+    @staticmethod
+    def _map_alba_nodes(api):
+        """
+        Will map the alba_node_id with its guid counterpart and return the map dict
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/backend.py]: add space between info and param.

File: ovs/extensions/healthcheck/helpers/storagerouter.py [hunk 14]
```diff
@@ -0,0 +1,150 @@
...
+        :return: list with master storagerouters ips
+        :rtype: list
+        """
+
+        return [storagerouter.ip for storagerouter in StorageRouterList.get_masters()]
+
+    @staticmethod
+    def get_by_machine_id(machine_id):
+        """
+        Fetch a dal machine by id
```
Comment (jeroenmaelbrancke) [inline ovs/extensions/healthcheck/helpers/storagerouter.py]: add space between info and param.

**Pull request merged**